src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs

```diff
@@ -35,6 +35,18 @@ public MvcOptions()
         }
 
         /// <summary>
+        /// Gets or sets the flag which decides whether input formatters (for example, using
+        /// <see cref="FromBodyAttribute"/>) should allow no input to be provided.
+        /// <see langword="false"/> by default.
```

Comment:
For descriptions like this, there's a tradeoff between technical accuracy and familiarity. It's more accurate to say "input formatters" because they are the ones that actually respond to the flag directly (whereas `BodyModelBinder` only responds to the result from the input formatters). So I erred on the side of accuracy in this case. But I appreciate that - as per your comment - most developers will interact with this system only on the level of model binding as a whole, so it's more familiar to phrase it in those terms. I think it's always going to be just a subjective call about how to pitch such a description. In this case I've rephrased in terms of "body model binding" ("model binding" alone would risk being too misleading, as this flag doesn't affect traditional `ComplexModelBinder` and friends).